### DISCUSSION TITLE
靈魂之窗受科技眷顧

### DISCUSSION THREAD

**Discussion by yi-1111** [1101-GS4719-C/Python-Community-GS4719-C#536] [科技新知]
就在去年，由澳洲雪梨大學和新南威爾斯大學的研究人員攜手進行了一項研究，他們為一群澳洲小羊做了外科手術，在小羊們的視網膜後方植入人工仿生眼，這項為期 3 個月的實驗目的是為了幫助研究人員開發出讓某類型失明患者恢復視力的新技術。

此次由雪梨大學和新南威爾斯大學共同開發的仿生眼「鳳凰99」，主要是以無線連接的方式配置在眼鏡上的小鏡頭上，再透過刺激用戶的視網膜來運作，也就是說，該技術將光轉換成電子信號，讓視網膜通過視神經發送到大腦，最後形成我們一般人所能看到的圖像。
會刻意繞開患者缺陷的視網膜神經細胞，進一步刺激那些仍存在著原先功能的細胞，而實驗過後，研究人員在設備的周圍也沒有發現到任何不適的現象，因此預期該裝置可以維持多年。

科技是否可以撬開上帝關閉的靈魂之窗，或許在不久的將來可以獲得答案

https://buzzorange.com/techorange/2022/03/23/eye-tech/

**Comment by Lynn12378** [comment]
這個技術可以幫助到很多需要的人，我覺得很棒

**Comment by Sazmafuji** [comment]
現在的人眼睛越來越不健康，可能我們未來罹患眼睛的疾病機率很高，此技術的進步，可能在我們之後會普遍用到

**Comment by ThomasMeng0515** [comment]
如果這項技術能成熟發展，可能會讓某些盲人重見天日，確實是一大福音

**Comment by ghost** [comment]
也希望能讓近視非常深的人有所受益

**Comment by sean1325418** [comment]
期待能永久解決近視問題的東西能快點出現

**Comment by Ku-Hao-Chen** [comment]
謝謝分享，希望科技能永久解決眼睛疾病的問題。

**Comment by PoooYen** [comment]
要實踐加上商用的話，還有10年以上的路要走吧，加油!

**Comment by HuanYuuuuuu** [comment]
不知道會不會有隱私權的問題，會被有心人士偷看你眼中的畫面

**Comment by henrywen547** [comment]
如果這項技術發展成熟，視障者也能如正常人一樣閱讀了!(也不用志工幫忙線上校對...，對，沒錯，就是我們服學做的事情...

**Comment by cordydeng** [comment]
這項技術如果能夠克服成本的問題，對於視障者來說一定是件好事

**Comment by Rosancl** [comment]
確實是對視障人士很棒的一項技術，但要等到價格變成大眾能負擔的起的時候仍需要時間。

**Comment by awkward-willy** [comment]
在眼睛裡植入電子眼聽起來有夠可怕的
不過若真的可行，想必能造福很多人
但若過程中有失敗或出差錯，可能會很難究責

**Comment by yenwei12138** [comment]
治療老花眼藥水真的很酷ㄟ，但看起來好像只能暫時減緩，而且還不清楚有沒有副作用
希望未來能多研發出像近視、閃光等眼藥水，也希望價格能夠親民些

**Comment by Lai27330** [comment]
希望未來技術成熟後能夠壓低價格，幫助到更多視障者。

**Comment by HaoFish123** [comment]
感謝分享，雖然這技術聽起來很酷，但好貴。

**Comment by treeann** [comment]
眼睛真的要從小好好保養....

**Comment by stan890425** [comment]
在商業化之前可能要先嘗試壓低成本

**Comment by jcpan821** [comment]
以後吃泡麵就不會霧霧了

**Comment by Ben6uo** [comment]
眼睛真的要好好保護 要不然就要花多很多錢錢了

**Comment by alexthebigpp** [comment]
有點cyberpunk的感覺了，不知道甚麼時候可以應用在人類身上。

**Comment by 110409538** [comment]
若能研發成功,且價格是一般民眾可以承擔的,將會是很大的貢獻.
